packages/trpc/server/routers/viewer/bookings/requestReschedule.handler.ts

```diff
@@ -198,17 +198,20 @@ export const requestRescheduleHandler = async ({ ctx, input }: RequestReschedule
   const bookingRefsFiltered: BookingReference[] = bookingToReschedule.references.filter((ref) =>
     credentialsMap.has(ref.type)
   );
-  const pCancellations = bookingRefsFiltered.map(async (bookingRef) => {
-    if (!bookingRef.uid) return;
 
-    if (bookingRef.type.endsWith("_calendar")) {
-      const calendar = await getCalendar(credentialsMap.get(bookingRef.type));
-      return calendar?.deleteEvent(bookingRef.uid, builder.calendarEvent, bookingRef.externalCalendarId);
-    } else if (bookingRef.type.endsWith("_video")) {
-      return deleteMeeting(credentialsMap.get(bookingRef.type), bookingRef.uid);
-    }
-  });
-  await Promise.all(pCancellations);
+  // FIXME: error-handling
+  await Promise.allSettled(
```

Comment:
Think this one is better suited for `allSettled`. Failure to delete a meeting should probably not prevent reschedule.